src/generated/cloud/compute/v1/Cargo.toml

```diff
@@ -254,6 +254,9 @@ zone-operations = []
 # Enables `client::Zones` and all the types it depends on.
 zones = []
 
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
Can you check the memory requirements? https://github.com/rust-lang/docs.rs/issues/3011 has some commands to get that info.

We may need to ask for more RAM.